index.html

```diff
@@ -0,0 +1,129 @@
+    </div>
+    <div>
+      <h2 class="date cocogoose">2015.10.15(THU) ~ 16(FRI)</h2>
+      <p class="loc lato">@National Museum of Korea. Art Center Nabi and more</p>
+    </div>
+  </div>
+  <section id="contents" class="contents d-flex gap-4">
+    <h1 class="content-header">Contents</h1>
+    <div class="tbl d-flex flex-md-row align-items-center gap-3">
+      <div class="row flex-lg-column row1"> 
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/book.svg" width="48" height="48" alt="book">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Reading Materials
+        </div>
+        <div class="col-6 col-md-12">
+          Offer references for reading materials to give context to your studies and to allow you to dig deep on your lessons.
+        </div>
+      </div>
+      <div class="row flex-md-column row2">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/play.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Supporting Contents
+        </div>
+        <div class="col-6 col-md-12">
+          To aid your studies, we have compiled a list of tutorials which we deem best from the web.
+        </div>
+      </div>
+       <div class="row flex-md-column row3">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/terminal.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Programming Tasks
+        </div>
+        <div class="col-6 col-md-12">
+          Coding tasks, mainly javascript function coding, along with the associative hints and answers.
+        </div>
+      </div>
+      <div class="row flex-md-column row3">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/terminal.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Programming Tasks
+        </div>
+        <div class="col-6 col-md-12">
+          Coding tasks, mainly javascript function coding, along with the associative hints and answers.
+        </div>
+      </div>
+      <div class="row flex-md-column row3">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/terminal.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Programming Tasks
+        </div>
+        <div class="col-6 col-md-12">
+          Coding tasks, mainly javascript function coding, along with the associative hints and answers.
+        </div>
+      </div>
+    </div>
+    <button type="button" class="sub m-3 mb-5 btn getstarted">Get started with the coding Challenges</button>
+  </section>
```

Comment:
### Your button
![your-button](https://user-images.githubusercontent.com/93039359/181395044-0d2e5561-c95d-4a23-b173-e1f059bbd341.png)
### Figma Template
![button](https://user-images.githubusercontent.com/93039359/181395061-7e9351ce-5fc8-41b1-bde5-e0f2d2272ffc.png)


- [ ] This `<button type="button" class="sub m-3 mb-5 btn getstarted">Get started with the coding Challenges</button>` should be displayed in a different style in the desktop version, but it will also appear in the desktop version with the same style as the mobile version. I think it would be better if you change the button to `hx` tag.